gnuradio-runtime/lib/flat_flowgraph.cc

```diff
@@ -77,10 +77,15 @@ namespace gr {
 
     // Connect message ports connetions
     for(msg_edge_viter_t i = d_msg_edges.begin(); i != d_msg_edges.end(); i++) {
-      if(FLAT_FLOWGRAPH_DEBUG)
-        std::cout << boost::format("flat_fg connecting msg primitives: (%s, %s)->(%s, %s)\n") %
+      if(FLAT_FLOWGRAPH_DEBUG) {
```

Comment:
There is no longer any need to put these in if() statements once the logging system is used, as the user can turn on/off debugging as needed in the config file.